components/brave_rewards/browser/rewards_p3a.h

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_REWARDS_BROWSER_REWARDS_P3A_H_
+#define BRAVE_COMPONENTS_BRAVE_REWARDS_BROWSER_REWARDS_P3A_H_
+
+#include <stddef.h>
+
+namespace brave_rewards {
+
+void RecordWalletBalanceP3A(bool wallet_created, size_t balance);
+
+enum class AutoContributionsP3AState {
```

Comment:
why this needs to be a enum class and not just a regular enum?

Reply:
using `enum class` is something default in modern C++